OKTA-803971 - Move hook best practice to Guides (#5072)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/hooks-best-practices/index.md

* Configure Okta to send an authentication header in the hook and validate it in every request by one of two ways:

>   * Using [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication). When activating and enabling your hooks on the Okta org, set the **Authorization field** as `Authorization` and the **Authentication secret** in the Base64-encoded `user:password` format.

Reviewer: ```suggestion
  * Using [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication). When activating and enabling hooks in an org, set the **Authorization field** to `Authorization`, and ensure that the **Authentication secret** is in the Base64-encoded `user:password` format.
```

## Protect your hook content from external viewers

> Okta requires HTTPS to encrypt communications to your hook endpoint. This requirement prevents unauthorized parties from reading the contents of an Okta hook. When using HTTPS, ensure you keep your SSL certificate updated and the Domain Name System (DNS) secured, so that someone can’t reroute your calls to another location.

Reviewer: ```suggestion
Okta requires HTTPS to encrypt communications to your hook endpoint. This requirement prevents unauthorized parties from reading the contents of a hook. When using HTTPS, ensure you keep your SSL certificate updated and the Domain Name System (DNS) secured, so that someone can’t reroute your calls to another location.
```
